include/freerdp/peer.h

```diff
@@ -54,13 +54,12 @@ typedef BOOL (*psPeerPostConnect)(freerdp_peer* peer);
 typedef BOOL (*psPeerActivate)(freerdp_peer* peer);
 typedef BOOL (*psPeerLogon)(freerdp_peer* peer, const SEC_WINNT_AUTH_IDENTITY* identity,
                             BOOL automatic);
-typedef BOOL (*psPeerSendServerRedirection)(freerdp_peer* peer, UINT32 sessionId,
-                                            const char* targetNetAddress, const char* routingToken,
-                                            const char* userName, const char* domain,
-                                            const char* password, const char* targetFQDN,
-                                            const char* targetNetBiosName, DWORD tsvUrlLength,
-                                            const BYTE* tsvUrl, UINT32 targetNetAddressesCount,
-                                            const char** targetNetAddresses);
+typedef BOOL (*psPeerSendServerRedirection)(
+    freerdp_peer* peer, UINT32 sessionId, const char* targetNetAddress, const char* routingToken,
+    const char* userName, const char* domain, const char* password, const char* targetFQDN,
```

Comment:
This should allow setting the `LB_PASSWORD_IS_PK_ENCRYPTED` flag.

Reply:
working on it, will push that later. (will make the `redirection_new` stuff public and use that as argument)